google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -203,5 +204,30 @@ public static StorageException toStorageException(Throwable exception) {
     return toStorageException(rpcStatus, exception);
   }
 
+  /**
+   * This exception is thrown from {@link JsonStreamWriter#append()} when the client side Json to
+   * Proto serializtion fails. The exception contains a Map of indexes of faulty lines and the
+   * corresponding error message.
+   */
+  public static class AppendSerializtionError extends RuntimeException {
+    private final Map<Integer, String> rowIndexToErrorMessage;
+    private final String streamName;
+
+    public AppendSerializtionError(
+        @Nullable String streamName, Map<Integer, String> rowIndexToErrorMessage) {
```

Comment:
Why Nullable?  I wouldn't expect getStreamName to ever return null

Reply:
done